packages/fxa-auth-server/lib/db/index.js

```diff
@@ -375,9 +312,8 @@ module.exports = (config, log, Token, UnblockCode = null) => {
 
     try {
       const [devices, redisSessionTokens = {}] = await Promise.all(promises);
-      const lastAccessTimeEnabled = features.isLastAccessTimeEnabledForUser(
-        uid
-      );
+      const lastAccessTimeEnabled =
```

Comment:
As a side note, the `lastAccessTime` feature has been around for a sometime now it seems and I wonder if we can graduate it.